src/components/controls/AudioMeter.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+export interface IAudioMeterRenderer {
+    (audioMonitor: any): JSX.Element | null | false;
+}
+
+export interface IAudioMeterProps {
+    audioMonitor: any;
+    render?: IAudioMeterRenderer
```

Comment:
I think we should remove this, since we don't encapsulate any business logic for the audio montior.

Reply:
Done